src/test/java/com/commercetools/sync/commons/utils/CustomUpdateActionUtilsTest.java

```diff
@@ -546,6 +546,22 @@ void buildNewOrModifiedCustomFieldsUpdateActions_WithNoNewOrModifiedCustomFields
         assertThat(updateActions).isEmpty();
     }
 
+    @Test
+    void buildNewOrModifiedCustomFieldsUpdateActions_WithNullCustomFields_ShouldNotBuildActions() {
+        final Map<String, JsonNode> oldCustomFields = new HashMap<>();
+        oldCustomFields.put("invisibleInShop", JsonNodeFactory.instance.booleanNode(true));
+        oldCustomFields.put("backgroundColor", JsonNodeFactory.instance.objectNode().put("de", "rot").put("en", "red"));
+
+        final Map<String, JsonNode> newCustomFields = new HashMap<>();
+        newCustomFields.put("invisibleInShop", null);
+
+        final List<UpdateAction<Product>> updateActions = buildNewOrModifiedCustomFieldsUpdateActions(oldCustomFields,
+            newCustomFields, mock(Price.class), new PriceCustomActionBuilder(), 1, Price::getId);
+
+        assertThat(updateActions).isNotNull();
```

Comment:
this assertion is not needed. It is already implicitly asserted in the following `isEmpty` assertion.

Reply:
fda7ecdcdb2b57ad5fec294465ebb86088c9f5f3